.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v6
+        with:
+          days-before-close: 30
+          days-before-stale: -1
+          stale-issue-label: status:needs-response
+          stale-pr-label: status:needs-response
+          close-issue-message: "This issue has been closed since a request for
+            information has not been answered for 30 days. It can be reopened
+            when the requested information is provided."
+          close-pr-message: "This PR has been closed since a request for
+            changes has not been answered for 30 days. It can be reopened when
+            the requested changes are provided."
```

Comment:
What if no one has requested anything, but the issue/PR author themselves indicated that they will add something later?

Reply:
Hmm, if the author said they will add more information and haven't, but we know the problem still exists and can still reproduce it, then I don't think the label should be used since we don't "need a response" to make progress on it.